newsletter/v208.0.1-February-2022.md

```diff
@@ -0,0 +1,118 @@
+  - `aria-*` props renamed to use aria naming convention
+  - `aria-label` added for a link (defaults to `type`)
+  - `altTag` renamed to `alt`
+- Logo
+  - `alt` defaults to `type`
+- FlashMessage
+  - `alert` role added
+- Avatar
+  - focus style is visible
+  - new props: `alt` & `ariaLinkLabel`
+- Icon
+  - added accessible `title` & `description` props for `Icon`
+  - added `role` to `svg` in `Icon` (default: `"img"`)
+  - adjusted colors for high contrast mode (only Windows)
+- AccordionItem
+  - added role `heading` and prop `headingLevel`
+
+Detailed accessibility documentation for each component in [Storybook](https://style-guide.brainly.com/) is coming soon.
+
+## Components color API changes
+
+Color API changes caused by introduction of new [color tokens](https://style-guide.brainly.com/?path=/story/foundation-%E2%9C%A8-colors--page).
+
+- Icon
+- MobileIcon
+- SubjectIcon
+- MathSymbol
+- IconAsButton
+- Text
+- TextBit
+- Link
+- Headline
+- Bubble
+- Box
+- Label
+- CardHole
+- Spinner, SpinnerContainer
+- Counter
+- Overlay
+- FileHandler
+- Media
+- Separators
+
+There's only `Button` component API left to be refactored, though the component itself is already consuming new color tokens.
+
+## New icons
+
+There was a bunch of new icons added in recent releases:
+
+- mobile
+  - circle_fill
+  - cursor_select
+  - dashed_line
+  - draw
+  - hand_move
+  - hexagon
+  - send_back
+  - send_front
+- desktop:
+  - bulb
+  - bulb_checked
+  - envelope
+  - keyboard
+  - profile_settings
+  - answers
+  - Google and Apple social icons
+
+## Responsive props
+
+Added responsive props support to following components:
+
+- Flex
+- Headline
+- Subheadline
+- Text
+- Link
+- Textbit
+
+Detailed documentation on the feature is coming soon, in the meantime please refer to these [release notes](https://github.com/brainly/style-guide/releases/tag/v202.1.0), where you can find the feature description.
+
+## UX motion
+
+Created UX motion tokens, including durations and easing functions. You can find recommended usage for each token [here](https://style-guide.brainly.com/?path=/story/foundation-%E2%9C%A8-ux-motion--page).
+
+Live examples of transitions are to be added in the future, for now please refer to [the Pencil documentation](https://design.brainly.com/8adfd5f36/p/091f75-ux-motion/b/5018fa) for visual representation of animations.
+
+## Product logotypes
+
+Introduced new [product logos and logotypes](https://style-guide.brainly.com/?path=/story/components-logo--product-logos).
+
+## Elevation in the UI
+
+Created [shadows utility classes](https://style-guide.brainly.com/?path=/story/utilities-shadows--page) and [documentation of elevation levels](https://style-guide.brainly.com/?path=/docs/foundation-%E2%9C%A8-shadows--page).
+
+## Dialog component
+
+New [Dialog](https://style-guide.brainly.com/?path=/story/components-dialog--default) component released.
+
+## Other
+
+- `Untitled` serif font face added (to be used in product only).
```

Comment:
And one more screenshot like this here? 😄  And information about Subheadline component using new font?

![Screenshot 2022-02-08 at 17 25 45](https://user-images.githubusercontent.com/25419830/153030769-744be720-4a39-4639-98ad-b8c671f7734e.png)